_posts/2020-06-30-sparklyr-1.3.0-released/sparklyr-1.3.0.Rmd

```diff
@@ -0,0 +1,202 @@
+---
+title: "sparklyr 1.3: Higher-order Functions, Avro, sparklyr.flint, EMR 6.0, Spark 3.0, and much more"
```

Comment:
I think we need to keep the title of a reasonable size, so I would scope it to:

```suggestion
title: "sparklyr 1.3: Higher-order Functions, Avro and Custom Serializers"
```

Also, it follows our tradition of showcasing the most relevant features that are being released in the blog post title.